app/src/main/kotlin/org/fossify/gallery/fragments/ViewPagerFragment.kt

```diff
@@ -65,7 +67,19 @@ abstract class ViewPagerFragment : Fragment() {
         }
 
         if (detailsFlag and EXT_RESOLUTION != 0) {
-            context!!.getResolution(file.absolutePath)?.formatAsResolution().let { if (it?.isNotEmpty() == true) details.appendLine(it) }
+            if(medium.name.endsWith(".jxl")) {
+                var jxlCoder = JxlCoder
+                var size = jxlCoder.getSize(file.readBytes())
```

Comment:
We can use JxlCoder directly. Also, it should be an immutable variable, so use `val` instead of `var`.

```suggestion
                val size = JxlCoder.getSize(file.readBytes())
```